web/pdf_page_view.js

```diff
@@ -78,12 +78,14 @@ import { XfaLayerBuilder } from "./xfa_layer_builder.js";
  * @property {number} [maxCanvasPixels] - The maximum supported canvas size in
  *   total pixels, i.e. width * height. Use `-1` for no limit, or `0` for
  *   CSS-only zooming. The default value is 4096 * 8192 (32 mega-pixels).
+ * @property {number} [maxCanvasDim] - The maximum supported canvas dimension,
+ *   in either width or height. Use `-1` for no limit.
```

Comment:
Looking at the code, it seems `0` also means "no limit"?

Reply:
That wasn't the intention, so I've update the patch to fix that; thanks for spotting the inconsistency.